### ISSUE TITLE
Release to ROS Noetic

### ISSUE THREAD

**Issue by wxmerkt** [ANYbotics/rqt_multiplot_plugin#44]
Hi, are there currently any plans or timeline to release this plugin for ROS Noetic?

**Comment by ovaag** [comment]
Bump

**Comment by EricGallimore** [comment]
I tried building this for Noetic.  I was able to get it to build (along with the three Variant packages, which I also had to clone into my workspace).  Variant generates a lot of warnings, but builds.

I can load the plugin in rqt, and everything looks good (I can configure the plot) until I start plotting.  Then, rqt crashes, and this message is printed:
python3: /ros/src/variant_topic_tools/src/BuiltinVariant.cpp:39: variant_topic_tools::BuiltinVariant::BuiltinVariant(const variant_topic_tools::Variant&): Assertion `boost::dynamic_pointer_cast<Value>(value)' failed.
